pyproject.toml

```diff
@@ -23,6 +23,9 @@ genisys = "genisys.main:main"
 
 [tool.poetry.dependencies]
 python = "^3.8"
+<<<<<<< HEAD
+PyYAML = "^5.1"
```

Comment:
This isn't showing as a merge conflict even though it looks like one? Fix this file with the correct requirements before we can merge

Reply:
I'm not sure if the change I pushed fixed the issue. Please look over the file to confirm.